swift/unowned-self-in-lazy-property.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+/// This playground checks to make sure that you don't need to do [unowned self] in a lazy var
```

Comment:
@noah Completely forgot about your PR! Mind updating this header to match the templates that have already been used?

Something like...
```
/**
 * GOAL:
 *    To determine if multiple calls to NSLayoutConstraint.activateConstraints() lead to a duplication of NSLayoutConstraint
 *    instances associated with their respective views.
 *
 * FINDINGS: 
 *    Multiple calls to NSLayoutConstraint.activateConstraints() does NOT lead to a duplication of NSLayoutConstraint instances
 *    associated with their respective views.
 *
 * Version: Apple Swift version 2.2 (swiftlang-703.0.18.5 clang-703.0.31)
 * Date:    4-28-2016
 * Author:  Grant Sheldon
 */
```